lens/util/repo.go

```diff
@@ -213,9 +204,9 @@ func MakeGetActorCodeFunc(ctx context.Context, store adt.Store, next, current *t
 			return cid.Undef, false
 		}
 
-		c, ok = actorCodes[ra]
-		if ok {
-			return c, true
+		act, err = nextStateTree.GetActor(ra)
+		if act != nil {
```

Comment:
```suggestion
		if err != nil {
```

Reply:
Same. Only returns when we have an actor object. Otherwise, use the previous state tree.